docker/.env.tpl

```diff
@@ -1,2 +1,3 @@
 BASEDIR=/srv/hapi/dev
-POSTGRES_PASSWORD=DEFAULT_POSTGRES_PASSWORD
\ No newline at end of file
+POSTGRES_PASSWORD=DEFAULT_POSTGRES_PASSWORD
+COMPOSE_PROJECT_NAME=hapi_stack
```

Comment:
@danmihaila @ccataalin please note the `COMPOSE_PROJECT_NAME=hapi_stack` in _env.tpl_. This will need to get into the .env file because I've removed the `name: hapi_stack` from the _docker-compose.yml_ file